geniza/corpus/management/commands/import_iiif_urls.py

```diff
@@ -0,0 +1,77 @@
+import csv
+import re
+import logging
+from collections import namedtuple
+
+from django.core.management.base import BaseCommand
+
+from geniza.corpus.models import Fragment
+
+
+# logging config: use levels as integers for verbosity option
+logger = logging.getLogger("import")
+logging.basicConfig()
+LOG_LEVELS = {0: logging.ERROR, 1: logging.WARNING, 2: logging.INFO, 3: logging.DEBUG}
+
+
+class Command(BaseCommand):
+    """Given a CSV of fragments and IIIF URLs, add those IIIF urls to their respective fragments in the database"""
+
+    def __init__(self, *args, **options):
+        self.csv_path = options.get("csv")
+        self.overwrite = options.get("overwrite")
+        self.dryrun = options.get("dryrun")
+
+    def add_arguments(self, parser):
+        parser.add_argument("-c", "--csv", type=str, required=True)
```

Comment:
since we always expect exactly one csv path, i'd consider this a positional argument rather than a flag